src/index.js

```diff
@@ -7,50 +7,95 @@ module.exports = {
       // default config
       {
         batchSize: 1,
-        insertOptions: { w: 1 },
+        operationType: 'insert'
```

Comment:
Here I have the feeling you break the API for some users.
Maybe we should keep `insertOptions` for now, and use it into `operation.insert`?